qiskit/circuit/library/__init__.py

```diff
@@ -85,15 +117,42 @@
     This summary table deliberately does not generate toctree entries; these directives are "owned"
     by ``qiskit.circuit``.
 
+Directives are operations to the quantum stack that are meant to be interpreted by the backend or
+the transpiler. In general, they transpiler or backend might optionally ignore them if there is no
+implementation for them.
+
 .. autosummary::
 
    ~qiskit.circuit.Barrier
+
+Standard Operations
+===================
+
+Operations are not reversible changes in the quantum state of the circuit.
+
+.. autosummary::
    ~qiskit.circuit.Measure
```

Comment:
This needs a blank line:

```suggestion
.. autosummary::
    :toctree: ../stubs/ 

   ~qiskit.circuit.Measure
```